### PR TITLE
Use logging config from kuksa-python-sdk

### PR DESCRIPTION
**Pull request by erikbosch**
Uses logging config from `kuksa-python-sdk`.

* Requires an updated kuksa-client pypi package to work and changed dependency.
* Also adapts to that Kuksa Python SDK no longer contains default tokens for webserver, so we must specify them explicitly
* Also updates dash dependency

### CODE REVIEW

File: dbcfeederlib/serverclientwrapper.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def start(self):
 
         self._kuksa = KuksaClientThread(self._client_config)
         self._kuksa.start()
-        self._kuksa.authorize()
+        self._kuksa.authorize(token_or_tokenfile=self._token_path)
```
Comment (erikbosch) [inline dbcfeederlib/serverclientwrapper.py:68]: Previously we did not consider what token was given in the config file, it worked anyway as kuksa-python-sdk had a default one which was the same one as we used in tests. This is an example on what we need to do to update client to latest kuksa-client. It should only affect VISS/websocket, but I assume we want to make sure thatw e support Websocket at least until end of year when Kuksa Server reaches end of life

**Comment by SebastianSchildt** [comment]
Some weird line breaks in log

image


Not sure it come from this PR, but do we want that?

**Comment by erikbosch** [comment]
It does not come from this PR, it is from this code: https://github.com/eclipse-kuksa/kuksa-can-provider/blob/main/dbcfeeder.py#L306

**Review by SebastianSchildt — APPROVED** [review]
Did some manual testing, looking fine.

lgtm 🍰

**Pull request merged**